docker-compose.yml

```diff
@@ -155,7 +155,7 @@ services:
       - SMTP_HOST=smtp4dev
       - SMTP_PASS=password
       - SMTP_PORT=25
-      - SMTP_USER=username
+      - SMTP_USER=username # TODO should it not be bob?
```

Comment:
I think this is the same for all personas as we only have one instance of the smtp4dev...but what do you think @mattdean-digicatapult ?

Reply:
Single instance.

Reply:
will remove a comment. thanks tho

Reply:
and it has been updated along with some usefull copmments e.g. on new functions in regards to partial query